client/components/foldable-card/index.js

```diff
@@ -0,0 +1,161 @@
+         }
+
+         if ( this.state.expanded ) {
+             this.props.onClose( this.props.cardKey );
+         } else {
+             this.props.onOpen( this.props.cardKey );
+         }
+     };
+
+     getClickAction() {
+         if ( this.props.disabled ) {
+             return;
+         }
+         return this.onClick;
+     }
+
+     getActionButton() {
+         if ( this.state.expanded ) {
+             return this.props.actionButtonExpanded || this.props.actionButton;
+         }
+         return this.props.actionButton;
+     }
+
+     renderActionButton() {
+         const clickAction = ! this.props.clickableHeader ? this.getClickAction() : null;
+         if ( this.props.actionButton ) {
+             return (
+                 <div className="foldable-card__action" role="presentation" onClick={ clickAction }>
+                     { this.getActionButton() }
+                 </div>
+             );
+         }
+         if ( this.props.children ) {
+             const iconSize = 24;
+             const screenReaderText = this.props.screenReaderText || this.props.translate( 'More' );
+             return (
+                 <button
+                     disabled={ this.props.disabled }
+                     type="button"
+                     className="foldable-card__action foldable-card__expand"
+                     onClick={ clickAction }
+                 >
+                     <ScreenReaderText>{ screenReaderText }</ScreenReaderText>
+                     <Gridicon icon={ this.props.icon } size={ iconSize } />
+                 </button>
+             );
+         }
+     }
+
+     renderContent() {
+         return <div className="foldable-card__content">{ this.props.children }</div>;
+     }
+
+     renderHeader() {
+         const summary = this.props.summary ? (
+             <span className="foldable-card__summary">{ this.props.summary } </span>
+         ) : null;
+         const expandedSummary = this.props.expandedSummary ? (
+             <span className="foldable-card__summary-expanded">{ this.props.expandedSummary } </span>
+         ) : null;
+         const headerClickAction = this.props.clickableHeader ? this.getClickAction() : null;
+         const headerClasses = classNames( 'foldable-card__header', {
+             'is-clickable': !! this.props.clickableHeader,
+             'has-border': !! this.props.summary,
+         } );
+         return (
+             <div className={ headerClasses } role="presentation" onClick={ headerClickAction }>
+                 <span className="foldable-card__main">{ this.props.header } </span>
+                 <span className="foldable-card__secondary">
+                     { summary }
+                     { expandedSummary }
+                     { this.renderActionButton() }
+                 </span>
+             </div>
+         );
+     }
+
+     render() {
+         const itemSiteClasses = classNames( 'foldable-card', this.props.className, {
```

Comment:
```suggestion
         const itemSiteClasses = classNames( 'foldable-card', 'card', this.props.className, {
```
The styles are a bit different when using the WordPress card. 
![NewFoldable](https://user-images.githubusercontent.com/8002881/124171168-d41ea280-da6d-11eb-8e9b-b09f80710994.png)
![CurrentFoldable](https://user-images.githubusercontent.com/8002881/124171187-d84ac000-da6d-11eb-9937-f962ad085dbc.png)


[On this PR](https://github.com/Automattic/woocommerce-services/pull/2337), we copied over the Calypso CSS for Card and then pass it to the WordPress Card. A bit redundant but it doesn't change the display.

Reply:
Thanks! Updated.